doc/source/reference/c-api/types-and-structures.rst

```diff
@@ -744,6 +745,16 @@ PyArrayMethod_Context and PyArrayMethod_Spec
       An array of descriptors for the ufunc loop, filled in by
       ``resolve_descriptors``. The length of the array is ``nin`` + ``nout``.
 
+   .. c:member:: void *parameters
+
+      A pointer to a structure containing any parameters needed by the
+      loop. This is ``NULL`` if no parameters are needed. The type of the
+      struct is specific to the loop.
+
+   .. versionchanged:: NumPy 2.4
+      The `parameters` member was added in NumPy 2.4 to allow passing
```

Comment:
I think no further explanation is needed. If we want to mention the sorting loop, might as well do it above, e.g., "It is used to pass flags to sort loops."